src/libraries/Microsoft.Extensions.FileProviders.Physical/src/PhysicalFilesWatcher.cs

```diff
@@ -466,7 +472,7 @@ internal static void RaiseChangeEvents(object state)
                 // We're already on a background thread, don't need to spawn a background Task to cancel the CTS
                 try
                 {
-                    token.CancellationTokenSource.Cancel();
+                    token.CancellationTokenSource?.Cancel();
```

Comment:
please exclude product changes - this most likely should be `!` anyway

Reply:
There's no guarantee (that I saw) that `CancellationTokenSource` is initialized. There's a property to to that, but it's not guaranteed to be called.

I'll change it to `!`, above is just FYI.